Source/Core/UICommon/X11Utils.cpp

```diff
@@ -103,7 +104,7 @@ XRRConfiguration::~XRRConfiguration()
 
 void XRRConfiguration::Update()
 {
-  if (SConfig::GetInstance().strFullscreenResolution == "Auto")
+  if (Config::Get(Config::MAIN_FULLSCREEN_DISPLAY_RES) == "Auto")
```

Comment:
I'm not completely sure how the Onion Config handles repeated calls, so it might be a good idea to cache the result of this call instead of doing it (up to) 3 times in here.